units/src/amount.rs

```diff
@@ -107,7 +107,7 @@ impl fmt::Display for Denomination {
 impl FromStr for Denomination {
     type Err = ParseDenominationError;
 
-    /// Convert from a str to Denomination.
+    /// Converts from a str to a Denomination.
```

Comment:
It'd be more appropriate to put the whole doc do on top of the trait impl, not method.

Reply:
I moved the section about Mega and accepted denominations to the top

Reply:
We should backtick `str` and `Denomination` I believe (or use "denomination").

Reply:
backticks added